sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -543,7 +543,7 @@
       "post": {
         "summary": "Create a new import/export job on an IoT hub.",
         "description": "Create a new import or export job on IoT Hub. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities for more information.",
-        "operationId": "JobClient_CreateImportExportJob",
+        "operationId": "Jobs_CreateImportExportJob",
```

Comment:
Job in the CreateImportExport**Job** is redundant.

Reply:
Yes we have redundant words in many of those. It'd be good to discuss this and device/modules changes together offline with team

Reply:
On second thought, I think CreateImportExport seems incomplete. Isn't removing adding some confusion?

Reply:
If I was on the service team, I would do two separate operations, one for Import and one for Export. Hence, two separate parameter types, ImportJobProperties and ExportJobProperties. 
Having both in a single operation makes it confusing on what properties of JobProperties are needed. Because Import takes in different properties than Export. But we're probably asking too much here from the service team as it's not a simple change.

Reply:
I agree to your point Basel. That would be a breaking change though!

Reply:
Our CL will cover up most of this anyways, so this current operationId is probably fine